daemon/guestos_mgr.c

```diff
@@ -105,16 +106,27 @@ guestos_mgr_load_operatingsystems_cb(const char *path, const char *name, UNUSED
 	case VERIFY_GOOD:
 		guestos_verified = GUESTOS_SIGNED;
 		INFO("Signature of GuestOS OK (GOOD)");
+		audit_log_event(uuid_new("00000000-0000-0000-0000-000000000000"), INFO, CMLD,
```

Comment:
mem_leek uuid_new(). (same for all other audit_log_event calls in this function).
Further it is discussable if ths event should be logged as it is triggerd from c0.
This is triggerd from CML itself during boot. Also guestos installation/updates may come from
other containers, too.